src/index.js

```diff
@@ -18,7 +18,11 @@ document.title = name;
 
 // concurrent mode
 const root = ReactDOM.createRoot(document.getElementById('app'));
-root.render(<App />);
+root.render(
+  <React.StrictMode>
```

Comment:
The React Profiler was complaining about StrictMode being missing, so I added it here. Not sure if you would prefer this in a separate PR.